test-e2e/cip-auditor/cip-auditor-e2e.go

```diff
@@ -554,21 +600,24 @@ func getCmdEmpowerServiceAccount(invokerServiceAccount string) []string {
 		fmt.Sprintf("--member=serviceAccount:%s", invokerServiceAccount),
 		"--role=roles/run.invoker",
 		"--platform=managed",
+		fmt.Sprintf("--project=%s", projectID),
+		"--region=us-central1",
 	}
 }
 
-func getCmdPurgePubSubMessages() []string {
+func getCmdPurgePubSubMessages(projectID string) []string {
 	return []string{
 		"gcloud",
 		"pubsub",
 		"subscriptions",
 		"seek",
 		subscriptionName,
 		"--time=+p1y",
+		fmt.Sprintf("--project=%s", projectID),
```

Comment:
Might be worth thinking about introducing a class to help with this, but ... can be done later.